lib/postProcessLdesConceptualService.js

```diff
@@ -392,3 +393,51 @@ async function flagInstancesModifiedConcept(vGraph, vService, service) {
 
   await updateSudo(updateQueryStr);
 }
+
+async function ensureConceptDisplayConfigs(versionedServiceGraph, versionedService, conceptualService) {
+  // This list limits the type of bestuurseenheden for which the config objects will be created.
+  // Only types that have access to the LPDC module should be added here.
+  const ALLOWED_BESTUURSEENHEID_CLASSIFICATIONS = [
```

Comment:
can you double check with business what the limitation is. Perhaps it would be easier to work with a blacklist.